src/virtualenv/create/via_global_ref/builtin/python2/python2.py

```diff
@@ -58,9 +58,11 @@ def sources(cls, interpreter):
         # install files needed to run site.py
         for req in cls.modules():
             stdlib_path = interpreter.stdlib_path("{}.py".format(req))
-            yield PathRefToDest(stdlib_path, dest=cls.to_stdlib)
             comp = stdlib_path.parent / "{}.pyc".format(req)
-            if comp.exists():
+            comp_exists = comp.exists()
+            if stdlib_path.exists() or not comp_exists:
+                yield PathRefToDest(stdlib_path, dest=cls.to_stdlib)
+            if comp_exists:
```

Comment:
Should this be an `elif`?

Reply:
Perhaps... The current behavior is to `yield` `.py` _and_ `.pyc` (if it exists), though, so I opted to preserve that.

Reply:
No, this should be two separate ifs. If the pyc exists copy it so the first load doesn't have to regenerate it.